source/connect-to-mongo/stable-api.txt

```diff
@@ -0,0 +1,134 @@
+.. _java-rs-stable-api:
+
+==========
+{+stable-api+}
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatible, backwards, upgrade
+
+.. note::
+
+   The {+stable-api+} feature requires {+mdb-server+} 5.0 or later.
+
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. When you update either your driver or server, 
+the API version changes, which can change the way these operations behave.
+Using the {+stable-api+} ensures consistent responses from the server and 
+provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``ServerApi`` object and specify a {+stable-api+} version. You must use
+   a {+stable-api+} version defined in the ``ServerApiVersion`` enum.
+#. Construct a ``MongoClientSettings`` object using the ``MongoClientSettings.Builder`` class.
+#. Instantiate a ``MongoClient`` using the ``MongoClients.create()`` method and 
+   pass your ``MongoClientSettings`` instance as a parameter.
+
+The following code example shows how to specify {+stable-api+} version 1:
+
+.. literalinclude:: /includes/connect/stable-api.java
+   :start-after: start-enable-stable-api
+   :end-before: end-enable-stable-api
+   :language: java
+   :copyable:
+   :dedent:
+
+Once you create a ``MongoClient`` instance with
+a specified API version, all commands you run with the client use the specified
+version. If you must run commands using more than one version of the 
```

Comment:
Currently there is only one version of the Stable API: V1.  

Consider changing this to refer to one of the other properties of the ServerApi class, e.g. `strict`.

Reply:
Do we expect another version to release? It could benefit us to cover our bases in advance of future versions.

I can also tweak the last highlighted sentence to the following more general version:

> If you must run Stable API commands using alternative configurations, create a new `MongoClient`.

Reply:
I have heard nothing about a new version coming out.